documentation/ame-guide/antora.yml

```diff
@@ -1,10 +1,10 @@
 name: ame-guide
 title: Aspect Model Editor
-version: "4.2.1"
+version: "main"
```

Comment:
This needs to be "master", which is a "magic keyword" in Antora 2, since we have [not yet switched](https://github.com/eclipse-esmf/eclipse-esmf.github.io/issues/4) to Antora 3.